packages/react/src/components/Toggle/index.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, boolean, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import { withInfo } from "@storybook/addon-info";
+
+import Toggle from "./";
+
+storiesOf("Buttons/Toggle", module)
+  .addDecorator(withKnobs)
+  .addDecorator(storiesFn => (
+    <div style={{ display: "flex", justifyContent: "center" }}>
+      {storiesFn()}
+    </div>
+  ))
+  .add(
+    "Playground",
+    withInfo("")(() => (
+      <Toggle
+        disabled={boolean("Is disabled?", false)}
+        label={text("Label", "Toggle label")}
+        helper={text("Label helper", "Label helper")}
+        onChange={state => action(`onChange - state: ${state}`)}
+      />
+    ))
+  )
+  .add("All states", () => (
+    <>
+      <div style={{ width: "100%" }}>
+        <Toggle label="Toggle" helper="Normal" />
```

Comment:
I'm having console errors for the `onChange` fn
https://www.dropbox.com/s/n3emevpjm9khzu2/Capture%20d%27%C3%A9cran%202019-08-12%2011.42.37.png?dl=0

Reply:
Fixed